### PR TITLE
if fade is true return next targetSlide on afterChange

### LINKED ISSUE #115: When fade:true afterChange returns wrong index
Hello, it's a weird one. ONLY if I set fade: true my afterChange function returns wrong index. I get 0, 1, 1, 2, 2, 3, 3..

So looks like I get previous index sometimes, maybe it is something to do with absence of webkitTransitionComplete with fade?

```
const carouselSettingsBig = {
      dots: false,
      infinite: true,
      speed: 500,
      fade: true,
      slidesToShow: 1,
      slidesToScroll: 1,
      lazyLoad: false,
      initialSlide: this.state.currentSlide,
      afterChange: function(index) {
        console.log('currentSlide', index)
        this.setState({ currentSlide: index });
      }.bind(this)
    };
```

### PR DESCRIPTION
**Pull request by Yuripetusko**
fixes #115

### CODE REVIEW

File: src/track.jsx [hunk 1]
```diff
@@ -16,19 +16,23 @@ var getSlideClasses = (spec) => {
...
+    if (index == spec.currentSlide - centerOffset + (spec.slidesToShow-1)) slickActiveLast = true;
+    if (index > spec.currentSlide - centerOffset - 1 && index <= spec.currentSlide + centerOffset) {
       slickActive = true;
     }
   } else {
-    slickActive = (spec.currentSlide <= index) && (index < spec.currentSlide + spec.slidesToShow);
+    slickActive = spec.currentSlide <= index && index < spec.currentSlide + spec.slidesToShow;
   }
-  return classnames({
+  return (0, _classnames2['default'])({
```
Comment (vramana) [inline src/track.jsx]: This is simply `classnames`. It seems you have copied this from a transpiled source. It doesn't work.

**Comment by Yuripetusko** [comment]
Sorry wrong PR! this is meant for my local branch

File: src/mixins/helpers.js [hunk 2]
```diff
@@ -110,7 +111,7 @@ var helpers = {
           animating: false
         });
         if (this.props.afterChange) {
-          this.props.afterChange(currentSlide);
+          this.props.afterChange(targetSlide);
```
Comment (Yuripetusko) [inline src/mixins/helpers.js]: @akiran I had only this line in originalPR, then i kept working on same branch and forgot that i created PR. sorry

**Comment by vramana** [comment]
@Yuripetusko Sorry for the wait. Ah!! Just resubmit the changes.

**Pull request closed without merging**